### ISSUE TITLE
Look into Turf.js for realtime geospatial analysis within ViziCities

### ISSUE THREAD

**Issue by robhawkes** [vizicities/vizicities#131]
A while ago Mapbox released [Turf.js](https://github.com/Turfjs/turf) and I didn't look into it enough to discover how useful it might be. It provides all sorts of standard geospatial analysis operations as well as other useful operations, like GDAL or QGIS in JavaScript basically.

Mapbox are using it for some cool visualisations, including [this 2D to 3D bike-route viewer](https://www.mapbox.com/bites/00094/) that I'm a little jealous of (it has a yummy transition).

TL;DR: Turf looks amazingly useful, we need to look into using it.
